packages/utils/src/instrument.ts

```diff
@@ -220,6 +223,9 @@ function instrumentXHR(): void {
     return;
   }
 
+  // Poor man implementation of ES6 `Map` by tracking and keeping in sync key and value separately.
```

Comment:
Could concurrent requests get this "map" out of sync?

Reply:
Nah, it's not Go 😅